### DISCUSSION TITLE
可以增加对mipsel架构处理器的支持吗

### DISCUSSION THREAD

**Discussion by RuiKylin** [zdz/ServerStatus-Rust#264] [General]
尝试用源码编译mipsel架构的版本均失败，看文档好像对于mipsel架构的支持还不太好。
因为不懂rust所以提出这个问题，目前是尝试编译用于 MT7621 处理器的版本。
请求帮助，谢谢！

**Comment by zdz** [comment]
我对 mipsel 不熟，再者用到的 sysinfo 库也没支持 OpenWrt

**Reply by RuiKylin** [reply]
是的我发现几个库暂时都不支持，不过client端有python版本的存在，也可以用其它语言实现。

感谢你开源和分享项目，谢谢！
